### PR TITLE
CommonBranch - Added allure steps, created new test class

### PR DESCRIPTION
**Pull request by VladimirPetrat**

### CODE REVIEW

File: src/test/java/com/softserveinc/ita/rozetka_test/RozetkaCategoryNavigationTests.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class RozetkaCategoryNavigationTests {
+
+    private LeftSidebar rozetkaLeftSidebar;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka_test/RozetkaCategoryNavigationTests.java]: drop rozetka in names

File: src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java [hunk 2]
```diff
@@ -1,45 +1,28 @@
...
 import static org.testng.Assert.*;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
-
-import static java.lang.String.format;
 
 public class RozetkaProductNavigationTest extends TestRunner {
 
-    private HomePage rozetkaHomePage;
+    private Header rozetkaHeader;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java]: same

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/page_objects/HomePage.java [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+        $x(format("(//li[contains(@class,'main-goods__cell')])[%d]", number)).click();
         return new ProductPage();
     }
 
-    public BasketPage openBasketPage() {
-        $x("//button[@class='header__button header__button--active']").click();
-        return new BasketPage();
+    public String getViewedProductName(int number) {
+        return $$x("//a[@class='tile__title']")
+                .shouldHave(CollectionCondition.sizeGreaterThanOrEqual(1))
```
Comment (SerhiiHladkyi) [inline src/main/java/com/softserveinc/ita/rozetka/page_objects/HomePage.java]: fix

File: src/main/java/com/softserveinc/ita/rozetka/utility_class/ShippingAddress.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package com.softserveinc.ita.rozetka.page_objects;
+package com.softserveinc.ita.rozetka.utility_class;
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/utility_class/ShippingAddress.java]: rename

File: src/test/java/com/softserveinc/ita/rozetka_test/CategoryNavigationTests.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class CategoryNavigationTests extends TestRunner {
+
+    private CatalogMenu catalogMenu;
+
+    @BeforeMethod
+    public void openHomepage() {
+        catalogMenu = new HomePage().openHomePage().getCatalogMenu();
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/rozetka_test/CategoryNavigationTests.java]: new lines

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/utility_class/ShippingAddress.java (hunk 4 above)
Comment (toleksyn) [reply src/main/java/com/softserveinc/ita/rozetka/utility_class/ShippingAddress.java]: models

**Pull request merged**